2012/10/03 12:12:04 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.Lietot훮jv훮rds" is not defined.' in /var/ludvus/yii/base/CComponent.php:131
Stack trace:
#0 /var/ludvus/yii/web/helpers/CHtml.php(2158): CComponent->__get('Lietot??jv??rds')
#1 /var/ludvus/yii/web/helpers/CHtml.php(1887): CHtml::resolveValue(Object(LoginForm), 'Lietot??jv??rds')
#2 /var/ludvus/yii/web/helpers/CHtml.php(1217): CHtml::activeInputField('text', Object(LoginForm), 'Lietot??jv??rds', Array)
#3 /var/ludvus/yii/web/widgets/CActiveForm.php(658): CHtml::activeTextField(Object(LoginForm), 'Lietot??jv??rds', Array)
#4 /var/ludvus/protected/views/site/login.php(23): CActiveForm->textField(Object(LoginForm), 'Lietot??jv??rds')
#5 /var/ludvus/yii/web/CBaseController.php(127): require('/var/ludvus/pro...')
#6 /var/ludvus/yii/web/CBaseController.php(96): CBaseController->renderInternal('/var/ludvus/pro...', Array, true)
#7 /var/ludvus/yii/web/CController.php(870): CBaseController->renderFile('/var/ludvus/pro...', Array, true)
#8 /var/ludvus/yii/web/CController.php(783): CController->renderPartial('login', Array, true)
#9 /var/ludvus/protected/controllers/SiteController.php(98): CController->render('login', Array)
#10 /var/ludvus/yii/web/actions/CInlineAction.php(50): SiteController->actionLogin()
#11 /var/ludvus/yii/web/CController.php(309): CInlineAction->runWithParams(Array)
#12 /var/ludvus/yii/web/CController.php(287): CController->runAction(Object(CInlineAction))
#13 /var/ludvus/yii/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 /var/ludvus/yii/web/CWebApplication.php(283): CController->run('login')
#15 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('site/login')
#16 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#17 /var/ludvus/index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://ludvus.local/index.php?r=site/contact
---
2012/10/03 12:12:05 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.Lietot훮jv훮rds" is not defined.' in /var/ludvus/yii/base/CComponent.php:131
Stack trace:
#0 /var/ludvus/yii/web/helpers/CHtml.php(2158): CComponent->__get('Lietot??jv??rds')
#1 /var/ludvus/yii/web/helpers/CHtml.php(1887): CHtml::resolveValue(Object(LoginForm), 'Lietot??jv??rds')
#2 /var/ludvus/yii/web/helpers/CHtml.php(1217): CHtml::activeInputField('text', Object(LoginForm), 'Lietot??jv??rds', Array)
#3 /var/ludvus/yii/web/widgets/CActiveForm.php(658): CHtml::activeTextField(Object(LoginForm), 'Lietot??jv??rds', Array)
#4 /var/ludvus/protected/views/site/login.php(23): CActiveForm->textField(Object(LoginForm), 'Lietot??jv??rds')
#5 /var/ludvus/yii/web/CBaseController.php(127): require('/var/ludvus/pro...')
#6 /var/ludvus/yii/web/CBaseController.php(96): CBaseController->renderInternal('/var/ludvus/pro...', Array, true)
#7 /var/ludvus/yii/web/CController.php(870): CBaseController->renderFile('/var/ludvus/pro...', Array, true)
#8 /var/ludvus/yii/web/CController.php(783): CController->renderPartial('login', Array, true)
#9 /var/ludvus/protected/controllers/SiteController.php(98): CController->render('login', Array)
#10 /var/ludvus/yii/web/actions/CInlineAction.php(50): SiteController->actionLogin()
#11 /var/ludvus/yii/web/CController.php(309): CInlineAction->runWithParams(Array)
#12 /var/ludvus/yii/web/CController.php(287): CController->runAction(Object(CInlineAction))
#13 /var/ludvus/yii/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 /var/ludvus/yii/web/CWebApplication.php(283): CController->run('login')
#15 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('site/login')
#16 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#17 /var/ludvus/index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://ludvus.local/index.php?r=site/login
---
2012/10/03 12:18:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "users/r".' in /var/ludvus/yii/web/CWebApplication.php:287
Stack trace:
#0 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('users/r')
#1 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#2 /var/ludvus/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=users/r
---
2012/10/03 12:18:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "users/r".' in /var/ludvus/yii/web/CWebApplication.php:287
Stack trace:
#0 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('users/r')
#1 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#2 /var/ludvus/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=users/r
HTTP_REFERER=http://ludvus.local/index.php?r=users/r
---
2012/10/03 12:18:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "users".' in /var/ludvus/yii/web/CWebApplication.php:287
Stack trace:
#0 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('users')
#1 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#2 /var/ludvus/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=users
---
2012/10/03 12:18:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "users".' in /var/ludvus/yii/web/CWebApplication.php:287
Stack trace:
#0 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('users')
#1 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#2 /var/ludvus/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=users
HTTP_REFERER=http://ludvus.local/index.php?r=users
---
2012/10/03 12:19:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "register".' in /var/ludvus/yii/web/CController.php:484
Stack trace:
#0 /var/ludvus/yii/web/CController.php(271): CController->missingAction('register')
#1 /var/ludvus/yii/web/CWebApplication.php(283): CController->run('register')
#2 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('site/register')
#3 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#4 /var/ludvus/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/register
---
2012/10/03 12:19:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "register".' in /var/ludvus/yii/web/CController.php:484
Stack trace:
#0 /var/ludvus/yii/web/CController.php(271): CController->missingAction('register')
#1 /var/ludvus/yii/web/CWebApplication.php(283): CController->run('register')
#2 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('site/register')
#3 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#4 /var/ludvus/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=site/register
HTTP_REFERER=http://ludvus.local/index.php?r=site/register
---
2012/10/03 12:20:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in /var/ludvus/yii/web/CWebApplication.php:287
Stack trace:
#0 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('gii')
#1 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#2 /var/ludvus/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=gii
---
2012/10/03 12:20:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in /var/ludvus/yii/web/CWebApplication.php:287
Stack trace:
#0 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('gii')
#1 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#2 /var/ludvus/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=gii
HTTP_REFERER=http://ludvus.local/index.php?r=gii
---
2012/10/03 12:35:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "users".' in /var/ludvus/yii/web/CWebApplication.php:287
Stack trace:
#0 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('users')
#1 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#2 /var/ludvus/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=users
---
2012/10/03 12:35:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "users".' in /var/ludvus/yii/web/CWebApplication.php:287
Stack trace:
#0 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('users')
#1 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#2 /var/ludvus/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=users
HTTP_REFERER=http://ludvus.local/index.php?r=users
---
2012/10/03 12:52:20 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.pass" is not defined.' in /var/ludvus/yii/base/CComponent.php:174
Stack trace:
#0 /var/ludvus/protected/controllers/SiteController.php(93): CComponent->__set('pass', 'da39a3ee5e6b4b0...')
#1 /var/ludvus/yii/web/actions/CInlineAction.php(50): SiteController->actionLogin()
#2 /var/ludvus/yii/web/CController.php(309): CInlineAction->runWithParams(Array)
#3 /var/ludvus/yii/web/CController.php(287): CController->runAction(Object(CInlineAction))
#4 /var/ludvus/yii/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 /var/ludvus/yii/web/CWebApplication.php(283): CController->run('login')
#6 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('site/login')
#7 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#8 /var/ludvus/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://ludvus.local/index.php?r=site/login
---
2012/10/03 13:01:40 [warning] [application] Failed to set unsafe attribute "username" of "LoginForm".
in /var/ludvus/protected/controllers/SiteController.php (92)
in /var/ludvus/index.php (13)
2012/10/03 13:01:40 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.username" is not defined.' in /var/ludvus/yii/base/CComponent.php:131
Stack trace:
#0 /var/ludvus/yii/web/helpers/CHtml.php(2158): CComponent->__get('username')
#1 /var/ludvus/yii/web/helpers/CHtml.php(1887): CHtml::resolveValue(Object(LoginForm), 'username')
#2 /var/ludvus/yii/web/helpers/CHtml.php(1217): CHtml::activeInputField('text', Object(LoginForm), 'username', Array)
#3 /var/ludvus/yii/web/widgets/CActiveForm.php(658): CHtml::activeTextField(Object(LoginForm), 'username', Array)
#4 /var/ludvus/protected/views/site/login.php(23): CActiveForm->textField(Object(LoginForm), 'username')
#5 /var/ludvus/yii/web/CBaseController.php(127): require('/var/ludvus/pro...')
#6 /var/ludvus/yii/web/CBaseController.php(96): CBaseController->renderInternal('/var/ludvus/pro...', Array, true)
#7 /var/ludvus/yii/web/CController.php(870): CBaseController->renderFile('/var/ludvus/pro...', Array, true)
#8 /var/ludvus/yii/web/CController.php(783): CController->renderPartial('login', Array, true)
#9 /var/ludvus/protected/controllers/SiteController.php(99): CController->render('login', Array)
#10 /var/ludvus/yii/web/actions/CInlineAction.php(50): SiteController->actionLogin()
#11 /var/ludvus/yii/web/CController.php(309): CInlineAction->runWithParams(Array)
#12 /var/ludvus/yii/web/CController.php(287): CController->runAction(Object(CInlineAction))
#13 /var/ludvus/yii/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 /var/ludvus/yii/web/CWebApplication.php(283): CController->run('login')
#15 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('site/login')
#16 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#17 /var/ludvus/index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://ludvus.local/index.php?r=site/login
---
2012/10/03 13:01:41 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.username" is not defined.' in /var/ludvus/yii/base/CComponent.php:131
Stack trace:
#0 /var/ludvus/yii/web/helpers/CHtml.php(2158): CComponent->__get('username')
#1 /var/ludvus/yii/web/helpers/CHtml.php(1887): CHtml::resolveValue(Object(LoginForm), 'username')
#2 /var/ludvus/yii/web/helpers/CHtml.php(1217): CHtml::activeInputField('text', Object(LoginForm), 'username', Array)
#3 /var/ludvus/yii/web/widgets/CActiveForm.php(658): CHtml::activeTextField(Object(LoginForm), 'username', Array)
#4 /var/ludvus/protected/views/site/login.php(23): CActiveForm->textField(Object(LoginForm), 'username')
#5 /var/ludvus/yii/web/CBaseController.php(127): require('/var/ludvus/pro...')
#6 /var/ludvus/yii/web/CBaseController.php(96): CBaseController->renderInternal('/var/ludvus/pro...', Array, true)
#7 /var/ludvus/yii/web/CController.php(870): CBaseController->renderFile('/var/ludvus/pro...', Array, true)
#8 /var/ludvus/yii/web/CController.php(783): CController->renderPartial('login', Array, true)
#9 /var/ludvus/protected/controllers/SiteController.php(99): CController->render('login', Array)
#10 /var/ludvus/yii/web/actions/CInlineAction.php(50): SiteController->actionLogin()
#11 /var/ludvus/yii/web/CController.php(309): CInlineAction->runWithParams(Array)
#12 /var/ludvus/yii/web/CController.php(287): CController->runAction(Object(CInlineAction))
#13 /var/ludvus/yii/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 /var/ludvus/yii/web/CWebApplication.php(283): CController->run('login')
#15 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('site/login')
#16 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#17 /var/ludvus/index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://ludvus.local/index.php?r=site/login
---
2012/10/03 13:02:59 [warning] [application] Failed to set unsafe attribute "username" of "LoginForm".
in /var/ludvus/protected/controllers/SiteController.php (92)
in /var/ludvus/index.php (13)
2012/10/03 13:02:59 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.username" is not defined.' in /var/ludvus/yii/base/CComponent.php:131
Stack trace:
#0 /var/ludvus/yii/web/helpers/CHtml.php(2158): CComponent->__get('username')
#1 /var/ludvus/yii/web/helpers/CHtml.php(1887): CHtml::resolveValue(Object(LoginForm), 'username')
#2 /var/ludvus/yii/web/helpers/CHtml.php(1217): CHtml::activeInputField('text', Object(LoginForm), 'username', Array)
#3 /var/ludvus/yii/web/widgets/CActiveForm.php(658): CHtml::activeTextField(Object(LoginForm), 'username', Array)
#4 /var/ludvus/protected/views/site/login.php(23): CActiveForm->textField(Object(LoginForm), 'username')
#5 /var/ludvus/yii/web/CBaseController.php(127): require('/var/ludvus/pro...')
#6 /var/ludvus/yii/web/CBaseController.php(96): CBaseController->renderInternal('/var/ludvus/pro...', Array, true)
#7 /var/ludvus/yii/web/CController.php(870): CBaseController->renderFile('/var/ludvus/pro...', Array, true)
#8 /var/ludvus/yii/web/CController.php(783): CController->renderPartial('login', Array, true)
#9 /var/ludvus/protected/controllers/SiteController.php(99): CController->render('login', Array)
#10 /var/ludvus/yii/web/actions/CInlineAction.php(50): SiteController->actionLogin()
#11 /var/ludvus/yii/web/CController.php(309): CInlineAction->runWithParams(Array)
#12 /var/ludvus/yii/web/CController.php(287): CController->runAction(Object(CInlineAction))
#13 /var/ludvus/yii/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 /var/ludvus/yii/web/CWebApplication.php(283): CController->run('login')
#15 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('site/login')
#16 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#17 /var/ludvus/index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://ludvus.local/index.php?r=site/login
---
2012/10/03 13:03:00 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.username" is not defined.' in /var/ludvus/yii/base/CComponent.php:131
Stack trace:
#0 /var/ludvus/yii/web/helpers/CHtml.php(2158): CComponent->__get('username')
#1 /var/ludvus/yii/web/helpers/CHtml.php(1887): CHtml::resolveValue(Object(LoginForm), 'username')
#2 /var/ludvus/yii/web/helpers/CHtml.php(1217): CHtml::activeInputField('text', Object(LoginForm), 'username', Array)
#3 /var/ludvus/yii/web/widgets/CActiveForm.php(658): CHtml::activeTextField(Object(LoginForm), 'username', Array)
#4 /var/ludvus/protected/views/site/login.php(23): CActiveForm->textField(Object(LoginForm), 'username')
#5 /var/ludvus/yii/web/CBaseController.php(127): require('/var/ludvus/pro...')
#6 /var/ludvus/yii/web/CBaseController.php(96): CBaseController->renderInternal('/var/ludvus/pro...', Array, true)
#7 /var/ludvus/yii/web/CController.php(870): CBaseController->renderFile('/var/ludvus/pro...', Array, true)
#8 /var/ludvus/yii/web/CController.php(783): CController->renderPartial('login', Array, true)
#9 /var/ludvus/protected/controllers/SiteController.php(99): CController->render('login', Array)
#10 /var/ludvus/yii/web/actions/CInlineAction.php(50): SiteController->actionLogin()
#11 /var/ludvus/yii/web/CController.php(309): CInlineAction->runWithParams(Array)
#12 /var/ludvus/yii/web/CController.php(287): CController->runAction(Object(CInlineAction))
#13 /var/ludvus/yii/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 /var/ludvus/yii/web/CWebApplication.php(283): CController->run('login')
#15 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('site/login')
#16 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#17 /var/ludvus/index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://ludvus.local/index.php?r=site/login
---
2012/10/03 13:19:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1044] Access denied for user 'ludvus'@'localhost' to database 'ludvs'' in /var/ludvus/yii/db/CDbConnection.php:382
Stack trace:
#0 /var/ludvus/yii/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/ludvus/yii/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/ludvus/yii/base/CModule.php(388): CDbConnection->init()
#3 /var/ludvus/yii/base/CApplication.php(431): CModule->getComponent('db')
#4 /var/ludvus/yii/db/ar/CActiveRecord.php(624): CApplication->getDb()
#5 /var/ludvus/yii/db/ar/CActiveRecord.php(2311): CActiveRecord->getDbConnection()
#6 /var/ludvus/yii/db/ar/CActiveRecord.php(388): CActiveRecordMetaData->__construct(Object(Users))
#7 /var/ludvus/yii/web/CActiveDataProvider.php(75): CActiveRecord::model('Users')
#8 /var/ludvus/protected/controllers/UserController.php(126): CActiveDataProvider->__construct('Users')
#9 /var/ludvus/yii/web/actions/CInlineAction.php(50): UserController->actionIndex()
#10 /var/ludvus/yii/web/CController.php(309): CInlineAction->runWithParams(Array)
#11 /var/ludvus/yii/web/filters/CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#12 /var/ludvus/yii/web/filters/CFilter.php(41): CFilterChain->run()
#13 /var/ludvus/yii/web/CController.php(1146): CFilter->filter(Object(CFilterChain))
#14 /var/ludvus/yii/web/filters/CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#15 /var/ludvus/yii/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#16 /var/ludvus/yii/web/CController.php(292): CFilterChain->run()
#17 /var/ludvus/yii/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /var/ludvus/yii/web/CWebApplication.php(283): CController->run('')
#19 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('user')
#20 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#21 /var/ludvus/index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/index.php?r=user
---
2012/10/03 13:19:21 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1044] Access denied for user 'ludvus'@'localhost' to database 'ludvs'' in /var/ludvus/yii/db/CDbConnection.php:382
Stack trace:
#0 /var/ludvus/yii/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/ludvus/yii/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/ludvus/yii/base/CModule.php(388): CDbConnection->init()
#3 /var/ludvus/yii/base/CApplication.php(431): CModule->getComponent('db')
#4 /var/ludvus/yii/db/ar/CActiveRecord.php(624): CApplication->getDb()
#5 /var/ludvus/yii/db/ar/CActiveRecord.php(2311): CActiveRecord->getDbConnection()
#6 /var/ludvus/yii/db/ar/CActiveRecord.php(388): CActiveRecordMetaData->__construct(Object(Users))
#7 /var/ludvus/yii/web/CActiveDataProvider.php(75): CActiveRecord::model('Users')
#8 /var/ludvus/protected/controllers/UserController.php(126): CActiveDataProvider->__construct('Users')
#9 /var/ludvus/yii/web/actions/CInlineAction.php(50): UserController->actionIndex()
#10 /var/ludvus/yii/web/CController.php(309): CInlineAction->runWithParams(Array)
#11 /var/ludvus/yii/web/filters/CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#12 /var/ludvus/yii/web/filters/CFilter.php(41): CFilterChain->run()
#13 /var/ludvus/yii/web/CController.php(1146): CFilter->filter(Object(CFilterChain))
#14 /var/ludvus/yii/web/filters/CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#15 /var/ludvus/yii/web/filters/CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#16 /var/ludvus/yii/web/CController.php(292): CFilterChain->run()
#17 /var/ludvus/yii/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /var/ludvus/yii/web/CWebApplication.php(283): CController->run('')
#19 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('user')
#20 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#21 /var/ludvus/index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/index.php?r=user
HTTP_REFERER=http://ludvus.local/index.php?r=user
---
2012/10/03 13:21:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in /var/ludvus/yii/web/CController.php:484
Stack trace:
#0 /var/ludvus/yii/web/CController.php(271): CController->missingAction('login')
#1 /var/ludvus/yii/web/CWebApplication.php(283): CController->run('login')
#2 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('user/login')
#3 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#4 /var/ludvus/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=user/login
---
2012/10/03 13:21:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in /var/ludvus/yii/web/CController.php:484
Stack trace:
#0 /var/ludvus/yii/web/CController.php(271): CController->missingAction('login')
#1 /var/ludvus/yii/web/CWebApplication.php(283): CController->run('login')
#2 /var/ludvus/yii/web/CWebApplication.php(142): CWebApplication->runController('user/login')
#3 /var/ludvus/yii/base/CApplication.php(162): CWebApplication->processRequest()
#4 /var/ludvus/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=user/login
HTTP_REFERER=http://ludvus.local/index.php?r=user/login
---
2012/10/03 15:28:55 [error] [exception.CDbException] exception 'CDbException' with message 'The table "users" for active record class "Users" cannot be found in the database.' in C:\xampp\htdocs\yii\db\ar\CActiveRecord.php:2312
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(388): CActiveRecordMetaData->__construct(Object(Users))
#1 C:\xampp\htdocs\protected\models\Users.php(22): CActiveRecord::model('Users')
#2 C:\xampp\htdocs\protected\components\UserIdentity.php(20): Users::model()
#3 C:\xampp\htdocs\protected\models\LoginForm.php(66): UserIdentity->authenticate()
#4 C:\xampp\htdocs\yii\validators\CInlineValidator.php(43): LoginForm->authenticate('password', Array)
#5 C:\xampp\htdocs\yii\validators\CValidator.php(214): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#6 C:\xampp\htdocs\yii\base\CModel.php(158): CValidator->validate(Object(LoginForm), NULL)
#7 C:\xampp\htdocs\protected\controllers\SiteController.php(95): CModel->validate()
#8 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#9 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('login')
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('site/login')
#14 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=site/login
HTTP_REFERER=http://localhost/index.php?r=site/login
---
2012/10/03 17:10:57 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.user_type" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CWebUser.php(141): CComponent->__get('user_type')
#1 C:\xampp\htdocs\protected\views\layouts\main.php(66): CWebUser->__get('user_type')
#2 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#6 C:\xampp\htdocs\yii\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#7 C:\xampp\htdocs\yii\web\CBaseController.php(207): COutputProcessor->run()
#8 C:\xampp\htdocs\yii\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#9 C:\xampp\htdocs\protected\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\yii\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\protected\controllers\SiteController.php(32): CController->render('index')
#14 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#15 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#17 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('')
#19 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('')
#20 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#21 C:\xampp\htdocs\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/index.php
HTTP_REFERER=http://localhost/index.php?r=site/login
---
2012/10/03 17:12:04 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\protected\views\layouts\main.php:66)
Stack trace:
#0 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(77): SiteController->renderFile()
#1 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate()
#2 C:\xampp\htdocs\yii\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput()
#3 C:\xampp\htdocs\yii\web\CBaseController.php(207): CContentDecorator->run()
#4 C:\xampp\htdocs\yii\web\CBaseController.php(301): SiteController->endWidget()
#5 C:\xampp\htdocs\protected\views\layouts\column1.php(6): SiteController->endContent()
#6 C:\xampp\htdocs\yii\web\CBaseController.php(127): require()
#7 C:\xampp\htdocs\yii\web\CBaseController.php(96): SiteController->renderInternal()
#8 C:\xampp\htdocs\yii\web\CController.php(785): SiteController->renderFile()
#9 C:\xampp\htdocs\protected\controllers\SiteController.php(32): SiteController->render()
#10 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#11 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#12 C:\xampp\htdocs\yii\web\CController.php(287): SiteController->runAction()
#13 C:\xampp\htdocs\yii\web\CController.php(266): SiteController->runActionWithFilters()
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(283): SiteController->run()
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#16 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#17 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php
in C:\xampp\htdocs\protected\views\layouts\main.php (66)
in C:\xampp\htdocs\protected\views\layouts\column1.php (6)
in C:\xampp\htdocs\protected\controllers\SiteController.php (32)
2012/10/03 17:13:26 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.password" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CWebUser.php(141): CComponent->__get('password')
#1 C:\xampp\htdocs\protected\views\layouts\main.php(66): CWebUser->__get('password')
#2 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#6 C:\xampp\htdocs\yii\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#7 C:\xampp\htdocs\yii\web\CBaseController.php(207): COutputProcessor->run()
#8 C:\xampp\htdocs\yii\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#9 C:\xampp\htdocs\protected\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\yii\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\protected\controllers\SiteController.php(32): CController->render('index')
#14 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#15 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#17 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('')
#19 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('')
#20 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#21 C:\xampp\htdocs\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/index.php
HTTP_REFERER=http://localhost/index.php?r=site/login
---
2012/10/03 17:13:31 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.pass" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CWebUser.php(141): CComponent->__get('pass')
#1 C:\xampp\htdocs\protected\views\layouts\main.php(66): CWebUser->__get('pass')
#2 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#6 C:\xampp\htdocs\yii\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#7 C:\xampp\htdocs\yii\web\CBaseController.php(207): COutputProcessor->run()
#8 C:\xampp\htdocs\yii\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#9 C:\xampp\htdocs\protected\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\yii\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\protected\controllers\SiteController.php(32): CController->render('index')
#14 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#15 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#17 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('')
#19 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('')
#20 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#21 C:\xampp\htdocs\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/index.php
HTTP_REFERER=http://localhost/index.php?r=site/login
---
2012/10/03 17:19:00 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.pass" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CWebUser.php(141): CComponent->__get('pass')
#1 C:\xampp\htdocs\protected\views\layouts\main.php(66): CWebUser->__get('pass')
#2 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#6 C:\xampp\htdocs\yii\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#7 C:\xampp\htdocs\yii\web\CBaseController.php(207): COutputProcessor->run()
#8 C:\xampp\htdocs\yii\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#9 C:\xampp\htdocs\protected\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\yii\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\protected\controllers\SiteController.php(32): CController->render('index')
#14 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#15 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#17 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('')
#19 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('')
#20 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#21 C:\xampp\htdocs\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/index.php
HTTP_REFERER=http://localhost/index.php?r=site/login
---
2012/10/03 17:27:00 [error] [exception.CException] exception 'CException' with message 'CWebUser and its behaviors do not have a method or closure named "getUserData".' in C:\xampp\htdocs\yii\base\CComponent.php:266
Stack trace:
#0 C:\xampp\htdocs\protected\views\layouts\main.php(66): CComponent->__call('getUserData', Array)
#1 C:\xampp\htdocs\protected\views\layouts\main.php(66): CWebUser->getUserData()
#2 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#6 C:\xampp\htdocs\yii\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#7 C:\xampp\htdocs\yii\web\CBaseController.php(207): COutputProcessor->run()
#8 C:\xampp\htdocs\yii\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#9 C:\xampp\htdocs\protected\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\yii\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\protected\controllers\SiteController.php(32): CController->render('index')
#14 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#15 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#17 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('')
#19 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('')
#20 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#21 C:\xampp\htdocs\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/index.php
HTTP_REFERER=http://localhost/index.php?r=site/login
---
2012/10/03 17:28:33 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.user_type" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CWebUser.php(141): CComponent->__get('user_type')
#1 C:\xampp\htdocs\protected\views\layouts\main.php(66): CWebUser->__get('user_type')
#2 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#6 C:\xampp\htdocs\yii\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#7 C:\xampp\htdocs\yii\web\CBaseController.php(207): COutputProcessor->run()
#8 C:\xampp\htdocs\yii\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#9 C:\xampp\htdocs\protected\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\yii\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\protected\controllers\SiteController.php(32): CController->render('index')
#14 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#15 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#17 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('')
#19 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('')
#20 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#21 C:\xampp\htdocs\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/index.php
HTTP_REFERER=http://localhost/index.php?r=site/login
---
2012/10/03 17:29:32 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.user_type" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CWebUser.php(141): CComponent->__get('user_type')
#1 C:\xampp\htdocs\protected\views\layouts\main.php(66): CWebUser->__get('user_type')
#2 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#6 C:\xampp\htdocs\yii\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#7 C:\xampp\htdocs\yii\web\CBaseController.php(207): COutputProcessor->run()
#8 C:\xampp\htdocs\yii\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#9 C:\xampp\htdocs\protected\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\yii\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\protected\controllers\SiteController.php(32): CController->render('index')
#14 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#15 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#17 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('')
#19 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('')
#20 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#21 C:\xampp\htdocs\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/
---
2012/10/03 17:30:05 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.user_type" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CWebUser.php(141): CComponent->__get('user_type')
#1 C:\xampp\htdocs\protected\views\layouts\main.php(67): CWebUser->__get('user_type')
#2 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#6 C:\xampp\htdocs\yii\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#7 C:\xampp\htdocs\yii\web\CBaseController.php(207): COutputProcessor->run()
#8 C:\xampp\htdocs\yii\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#9 C:\xampp\htdocs\protected\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\yii\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\protected\controllers\SiteController.php(32): CController->render('index')
#14 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#15 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#17 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('')
#19 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('')
#20 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#21 C:\xampp\htdocs\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/
---
2012/10/03 17:30:07 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.user_type" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CWebUser.php(141): CComponent->__get('user_type')
#1 C:\xampp\htdocs\protected\views\layouts\main.php(67): CWebUser->__get('user_type')
#2 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(77): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\web\widgets\CContentDecorator.php(55): CContentDecorator->decorate('<div id="conten...')
#6 C:\xampp\htdocs\yii\web\widgets\COutputProcessor.php(45): CContentDecorator->processOutput('<div id="conten...')
#7 C:\xampp\htdocs\yii\web\CBaseController.php(207): COutputProcessor->run()
#8 C:\xampp\htdocs\yii\web\CBaseController.php(301): CBaseController->endWidget('CContentDecorat...')
#9 C:\xampp\htdocs\protected\views\layouts\column1.php(6): CBaseController->endContent()
#10 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\yii\web\CController.php(785): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\protected\controllers\SiteController.php(32): CController->render('index')
#14 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#15 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#16 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#17 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('')
#19 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('')
#20 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#21 C:\xampp\htdocs\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/
---
2012/10/03 18:18:39 [error] [exception.CException] exception 'CException' with message 'Property "Jaunumi.title" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(144): CComponent->__get('title')
#1 C:\xampp\htdocs\yii\base\CModel.php(593): CActiveRecord->__get('title')
#2 C:\xampp\htdocs\protected\views\site\index.php(13): CModel->offsetGet('title')
#3 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\web\CController.php(870): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\web\CController.php(783): CController->renderPartial('index', Array, true)
#7 C:\xampp\htdocs\protected\controllers\SiteController.php(31): CController->render('index', Array)
#8 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#9 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('')
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('')
#14 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php
---
2012/10/03 18:26:08 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\yii\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(310): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\web\filters\CFilter.php(41): CFilterChain->run()
#3 C:\xampp\htdocs\yii\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('view')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user/view')
#10 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=user/view
HTTP_REFERER=http://localhost/index.php
---
2012/10/03 18:27:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\protected\controllers\UserController.php:156
Stack trace:
#0 C:\xampp\htdocs\protected\controllers\UserController.php(55): UserController->loadModel('komandants')
#1 [internal function]: UserController->actionView('komandants')
#2 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(UserController), Array)
#3 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\CController.php(292): CFilterChain->run()
#11 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('view')
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user/view')
#14 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=user/view&id=komandants
HTTP_REFERER=http://localhost/index.php?r=user/view&id=1
---
2012/10/03 18:28:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\protected\controllers\UserController.php:156
Stack trace:
#0 C:\xampp\htdocs\protected\controllers\UserController.php(55): UserController->loadModel('komandants')
#1 [internal function]: UserController->actionView('komandants')
#2 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(UserController), Array)
#3 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\CController.php(292): CFilterChain->run()
#11 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('view')
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user/view')
#14 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=user/view&id=komandants
HTTP_REFERER=http://localhost/index.php?r=user/view&id=1
---
2012/10/03 18:31:40 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\web\auth\CAccessControlFilter.php:170
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CAccessControlFilter.php(133): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\CController.php(292): CFilterChain->run()
#6 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('update')
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user/update')
#9 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/update&id=2
HTTP_REFERER=http://localhost/index.php?r=user/view&id=2
---
2012/10/03 18:41:16 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\web\auth\CAccessControlFilter.php:170
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CAccessControlFilter.php(133): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\CController.php(292): CFilterChain->run()
#6 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('update')
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user/update')
#9 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/update&id=2
HTTP_REFERER=http://localhost/index.php?r=user/view&id=2
---
2012/10/03 18:45:03 [error] [exception.CException] exception 'CException' with message 'Property "Users.vards" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(144): CComponent->__get('vards')
#1 C:\xampp\htdocs\yii\web\helpers\CHtml.php(2158): CActiveRecord->__get('vards')
#2 C:\xampp\htdocs\yii\web\helpers\CHtml.php(1887): CHtml::resolveValue(Object(Users), 'vards')
#3 C:\xampp\htdocs\yii\web\helpers\CHtml.php(1217): CHtml::activeInputField('text', Object(Users), 'vards', Array)
#4 C:\xampp\htdocs\yii\web\widgets\CActiveForm.php(658): CHtml::activeTextField(Object(Users), 'vards', Array)
#5 C:\xampp\htdocs\protected\views\user\iemitnieki.php(20): CActiveForm->textField(Object(Users), 'vards')
#6 C:\xampp\htdocs\yii\web\CBaseController.php(127): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\yii\web\CBaseController.php(96): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\web\CController.php(870): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii\web\CController.php(783): CController->renderPartial('iemitnieki', Array, true)
#10 C:\xampp\htdocs\protected\controllers\UserController.php(73): CController->render('iemitnieki', Array)
#11 [internal function]: UserController->actionView('1')
#12 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(UserController), Array)
#13 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#14 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\web\filters\CFilter.php(41): CFilterChain->run()
#17 C:\xampp\htdocs\yii\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#19 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\yii\web\CController.php(292): CFilterChain->run()
#21 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('view')
#23 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user/view')
#24 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#25 C:\xampp\htdocs\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/index.php?r=user/view&id=1
HTTP_REFERER=http://localhost/index.php
---
2012/10/03 18:55:17 [error] [exception.CException] exception 'CException' with message 'Property "Users.id" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(144): CComponent->__get('id')
#1 C:\xampp\htdocs\protected\controllers\UserController.php(75): CActiveRecord->__get('id')
#2 [internal function]: UserController->actionView('1')
#3 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(UserController), Array)
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\filters\CFilter.php(41): CFilterChain->run()
#8 C:\xampp\htdocs\yii\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\CController.php(292): CFilterChain->run()
#12 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('view')
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user/view')
#15 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#16 C:\xampp\htdocs\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/index.php?r=user/view&id=1
HTTP_REFERER=http://localhost/index.php?r=user/view&id=1
---
2012/10/03 18:56:08 [error] [php] Undefined variable: render (C:\xampp\htdocs\protected\controllers\UserController.php:78)
Stack trace:
#0 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CInlineAction->runWithParamsInternal()
#1 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(134): UserController->runAction()
#3 C:\xampp\htdocs\yii\web\filters\CFilter.php(41): CFilterChain->run()
#4 C:\xampp\htdocs\yii\web\CController.php(1146): CAccessControlFilter->filter()
#5 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): UserController->filterAccessControl()
#6 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#7 C:\xampp\htdocs\yii\web\CController.php(292): CFilterChain->run()
#8 C:\xampp\htdocs\yii\web\CController.php(266): UserController->runActionWithFilters()
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(283): UserController->run()
#10 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#11 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#12 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=user/view&id=2
in C:\xampp\htdocs\protected\controllers\UserController.php (78)
in C:\xampp\htdocs\index.php (13)
2012/10/03 20:27:52 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'ludvus'@'localhost' (using password: YES)' in C:\xampp\htdocs\yii\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\yii\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\yii\db\CDbConnection.php(309): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\base\CModule.php(388): CDbConnection->init()
#3 C:\xampp\htdocs\yii\base\CApplication.php(431): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(624): CApplication->getDb()
#5 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(2311): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(388): CActiveRecordMetaData->__construct(Object(Jaunumi))
#7 C:\xampp\htdocs\protected\models\Jaunumi.php(22): CActiveRecord::model('Jaunumi')
#8 C:\xampp\htdocs\protected\controllers\SiteController.php(34): Jaunumi::model()
#9 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#10 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('')
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('')
#15 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#16 C:\xampp\htdocs\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/
---
2012/10/03 20:30:13 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'ludvus.jaunumi' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `jaunumi`.
in C:\xampp\htdocs\protected\models\Jaunumi.php (22)
in C:\xampp\htdocs\protected\controllers\SiteController.php (34)
in C:\xampp\htdocs\index.php (13)
2012/10/03 20:30:13 [error] [exception.CDbException] exception 'CDbException' with message 'The table "jaunumi" for active record class "Jaunumi" cannot be found in the database.' in C:\xampp\htdocs\yii\db\ar\CActiveRecord.php:2312
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(388): CActiveRecordMetaData->__construct(Object(Jaunumi))
#1 C:\xampp\htdocs\protected\models\Jaunumi.php(22): CActiveRecord::model('Jaunumi')
#2 C:\xampp\htdocs\protected\controllers\SiteController.php(34): Jaunumi::model()
#3 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionIndex()
#4 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('')
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('')
#9 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/
---
2012/10/04 08:57:52 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "administratori_form".' in C:\xampp\htdocs\yii\web\CController.php:879
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(783): CController->renderPartial('administratori_...', Array, true)
#1 C:\xampp\htdocs\protected\controllers\SiteController.php(73): CController->render('administratori_...', Array)
#2 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionProfile()
#3 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('profile')
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('site/profile')
#8 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\xampp\htdocs\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=site/profile
---
2012/10/04 11:12:05 [error] [php] Undefined variable: render (C:\xampp\htdocs\protected\controllers\UserController.php:78)
Stack trace:
#0 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CInlineAction->runWithParamsInternal()
#1 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(134): UserController->runAction()
#3 C:\xampp\htdocs\yii\web\filters\CFilter.php(41): CFilterChain->run()
#4 C:\xampp\htdocs\yii\web\CController.php(1146): CAccessControlFilter->filter()
#5 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): UserController->filterAccessControl()
#6 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#7 C:\xampp\htdocs\yii\web\CController.php(292): CFilterChain->run()
#8 C:\xampp\htdocs\yii\web\CController.php(266): UserController->runActionWithFilters()
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(283): UserController->run()
#10 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#11 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#12 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=user/view&id=2
in C:\xampp\htdocs\protected\controllers\UserController.php (78)
in C:\xampp\htdocs\index.php (13)
2012/10/04 11:39:13 [error] [php] Undefined index: Iemitnieki (C:\xampp\htdocs\protected\controllers\NewsController.php:20)
Stack trace:
#0 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CInlineAction->runWithParamsInternal()
#1 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\web\CController.php(287): NewsController->runAction()
#3 C:\xampp\htdocs\yii\web\CController.php(266): NewsController->runActionWithFilters()
#4 C:\xampp\htdocs\yii\web\CWebApplication.php(283): NewsController->run()
#5 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#6 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#7 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=news/edit&id=2
in C:\xampp\htdocs\protected\controllers\NewsController.php (20)
in C:\xampp\htdocs\index.php (13)
2012/10/04 11:49:22 [warning] [application] Failed to set unsafe attribute "ts" of "Jaunumi".
in C:\xampp\htdocs\protected\controllers\NewsController.php (34)
in C:\xampp\htdocs\index.php (13)
2012/10/04 12:36:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in C:\xampp\htdocs\yii\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user')
#1 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user
---
2012/10/04 13:01:26 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Jums nav ties카bu apmekl휆t 큄o lapu' in C:\xampp\htdocs\protected\components\Controller.php:27
Stack trace:
#0 C:\xampp\htdocs\protected\controllers\NewsController.php(7): Controller->allowUser(2)
#1 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): NewsController->actionIndex()
#2 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('')
#6 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('news')
#7 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\xampp\htdocs\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/index.php?r=news
---
2012/10/04 13:02:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in C:\xampp\htdocs\yii\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user')
#1 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user
---
2012/10/04 13:02:37 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.user_type" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CWebUser.php(141): CComponent->__get('user_type')
#1 C:\xampp\htdocs\protected\controllers\SiteController.php(57): CWebUser->__get('user_type')
#2 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionProfile()
#3 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('profile')
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('site/profile')
#8 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\xampp\htdocs\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=site/profile
---
2012/10/04 13:02:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in C:\xampp\htdocs\yii\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user')
#1 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user
---
2012/10/04 13:06:23 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.user_type" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CWebUser.php(141): CComponent->__get('user_type')
#1 C:\xampp\htdocs\protected\controllers\SiteController.php(74): CWebUser->__get('user_type')
#2 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionProfile()
#3 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('profile')
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('site/profile')
#8 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\xampp\htdocs\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=site/profile
---
2012/10/04 13:06:44 [error] [exception.CException] exception 'CException' with message 'Property "CWebUser.user_type" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CWebUser.php(141): CComponent->__get('user_type')
#1 C:\xampp\htdocs\protected\controllers\SiteController.php(74): CWebUser->__get('user_type')
#2 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionProfile()
#3 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('profile')
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('site/profile')
#8 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\xampp\htdocs\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/index.php?r=site/profile
---
2012/10/04 13:09:20 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Jums nav ties카bu apmekl휆t 큄o lapu' in C:\xampp\htdocs\protected\components\Controller.php:27
Stack trace:
#0 C:\xampp\htdocs\protected\controllers\SiteController.php(57): Controller->allowUser(1)
#1 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionProfile()
#2 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('profile')
#6 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('site/profile')
#7 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\xampp\htdocs\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/index.php?r=site/profile
---
2012/10/04 13:52:17 [error] [php] Missing argument 3 for CActiveForm::dropDownList(), called in C:\xampp\htdocs\protected\views\site\iemitnieki_form.php on line 38 and defined (C:\xampp\htdocs\yii\web\widgets\CActiveForm.php:762)
Stack trace:
#0 C:\xampp\htdocs\yii\web\CBaseController.php(96): SiteController->renderInternal()
#1 C:\xampp\htdocs\yii\web\CController.php(870): SiteController->renderFile()
#2 C:\xampp\htdocs\yii\web\CController.php(783): SiteController->renderPartial()
#3 C:\xampp\htdocs\protected\controllers\SiteController.php(95): SiteController->render()
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionProfile()
#5 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#6 C:\xampp\htdocs\yii\web\CController.php(287): SiteController->runAction()
#7 C:\xampp\htdocs\yii\web\CController.php(266): SiteController->runActionWithFilters()
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(283): SiteController->run()
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#10 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/profile
in C:\xampp\htdocs\protected\views\site\iemitnieki_form.php (38)
in C:\xampp\htdocs\protected\controllers\SiteController.php (95)
in C:\xampp\htdocs\index.php (13)
2012/10/04 13:53:17 [error] [php] Object of class Fakultates could not be converted to string (C:\xampp\htdocs\yii\web\helpers\CHtml.php:1978)
Stack trace:
#0 C:\xampp\htdocs\protected\views\site\iemitnieki_form.php(38): CActiveForm->dropDownList()
#1 C:\xampp\htdocs\yii\web\CBaseController.php(127): require()
#2 C:\xampp\htdocs\yii\web\CBaseController.php(96): SiteController->renderInternal()
#3 C:\xampp\htdocs\yii\web\CController.php(870): SiteController->renderFile()
#4 C:\xampp\htdocs\yii\web\CController.php(783): SiteController->renderPartial()
#5 C:\xampp\htdocs\protected\controllers\SiteController.php(95): SiteController->render()
#6 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionProfile()
#7 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\web\CController.php(287): SiteController->runAction()
#9 C:\xampp\htdocs\yii\web\CController.php(266): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\web\CWebApplication.php(283): SiteController->run()
#11 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#13 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/profile
in C:\xampp\htdocs\protected\views\site\iemitnieki_form.php (38)
in C:\xampp\htdocs\protected\controllers\SiteController.php (95)
in C:\xampp\htdocs\index.php (13)
2012/10/04 14:01:27 [error] [php] htmlspecialchars() expects parameter 1 to be string, array given (C:\xampp\htdocs\yii\web\helpers\CHtml.php:85)
Stack trace:
#0 C:\xampp\htdocs\yii\web\helpers\CHtml.php(140): renderAttributes()
#1 C:\xampp\htdocs\yii\web\helpers\CHtml.php(1525): tag()
#2 C:\xampp\htdocs\yii\web\widgets\CActiveForm.php(764): activeDropDownList()
#3 C:\xampp\htdocs\protected\views\site\iemitnieki_form.php(38): CActiveForm->dropDownList()
#4 C:\xampp\htdocs\yii\web\CBaseController.php(127): require()
#5 C:\xampp\htdocs\yii\web\CBaseController.php(96): SiteController->renderInternal()
#6 C:\xampp\htdocs\yii\web\CController.php(870): SiteController->renderFile()
#7 C:\xampp\htdocs\yii\web\CController.php(783): SiteController->renderPartial()
#8 C:\xampp\htdocs\protected\controllers\SiteController.php(96): SiteController->render()
#9 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionProfile()
#10 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#11 C:\xampp\htdocs\yii\web\CController.php(287): SiteController->runAction()
#12 C:\xampp\htdocs\yii\web\CController.php(266): SiteController->runActionWithFilters()
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(283): SiteController->run()
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#15 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#16 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/profile
in C:\xampp\htdocs\protected\views\site\iemitnieki_form.php (38)
in C:\xampp\htdocs\protected\controllers\SiteController.php (96)
in C:\xampp\htdocs\index.php (13)
2012/10/04 14:10:36 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\web\auth\CAccessControlFilter.php:170
Stack trace:
#0 C:\xampp\htdocs\yii\web\auth\CAccessControlFilter.php(133): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\web\CController.php(292): CFilterChain->run()
#6 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('admin')
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user/admin')
#9 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php?r=user/admin
HTTP_REFERER=http://localhost/index.php?r=user/index
---
2012/10/04 14:18:19 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Jums nav ties카bu apmekl휆t 큄o lapu' in C:\xampp\htdocs\protected\components\Controller.php:29
Stack trace:
#0 C:\xampp\htdocs\protected\controllers\UserController.php(41): Controller->allowUser(3)
#1 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): UserController->actionCreate()
#2 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii\web\filters\CFilter.php(41): CFilterChain->run()
#5 C:\xampp\htdocs\yii\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\web\CController.php(292): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('create')
#11 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user/create')
#12 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#13 C:\xampp\htdocs\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/index.php?r=user/create
HTTP_REFERER=http://localhost/index.php?r=user/view&id=1
---
2012/10/04 14:18:25 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Jums nav ties카bu apmekl휆t 큄o lapu' in C:\xampp\htdocs\protected\components\Controller.php:29
Stack trace:
#0 C:\xampp\htdocs\protected\controllers\UserController.php(41): Controller->allowUser(3)
#1 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): UserController->actionCreate()
#2 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii\web\filters\CFilter.php(41): CFilterChain->run()
#5 C:\xampp\htdocs\yii\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\web\CController.php(292): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('create')
#11 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user/create')
#12 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#13 C:\xampp\htdocs\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/index.php?r=user/create
HTTP_REFERER=http://localhost/index.php?r=user/view&id=1
---
2012/10/04 14:18:32 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Jums nav ties카bu apmekl휆t 큄o lapu' in C:\xampp\htdocs\protected\components\Controller.php:29
Stack trace:
#0 C:\xampp\htdocs\protected\controllers\UserController.php(93): Controller->allowUser(3)
#1 [internal function]: UserController->actionDelete('1')
#2 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(UserController), Array)
#3 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\web\CController.php(1117): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): CController->filterPostOnly(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\web\filters\CFilter.php(41): CFilterChain->run()
#10 C:\xampp\htdocs\yii\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\yii\web\CController.php(292): CFilterChain->run()
#14 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('delete')
#16 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user/delete')
#17 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#18 C:\xampp\htdocs\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/index.php?r=user/delete&id=1
HTTP_REFERER=http://localhost/index.php?r=user/view&id=1
---
2012/10/04 14:18:36 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Jums nav ties카bu apmekl휆t 큄o lapu' in C:\xampp\htdocs\protected\components\Controller.php:29
Stack trace:
#0 C:\xampp\htdocs\protected\controllers\UserController.php(120): Controller->allowUser(3)
#1 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): UserController->actionAdmin()
#2 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii\web\filters\CFilter.php(41): CFilterChain->run()
#5 C:\xampp\htdocs\yii\web\CController.php(1146): CFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\web\CController.php(292): CFilterChain->run()
#9 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('admin')
#11 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('user/admin')
#12 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#13 C:\xampp\htdocs\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/index.php?r=user/admin
HTTP_REFERER=http://localhost/index.php?r=user/view&id=1
---
2012/10/05 15:23:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "pieteikumi/view".' in C:\xampp\htdocs\yii\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('pieteikumi/view')
#1 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=pieteikumi/view
---
2012/10/05 15:23:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "pieteikumi/view".' in C:\xampp\htdocs\yii\web\CWebApplication.php:287
Stack trace:
#0 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('pieteikumi/view')
#1 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=pieteikumi/view
HTTP_REFERER=http://localhost/index.php?r=pieteikumi/view
---
2012/10/05 15:23:41 [error] [php] Undefined variable: k (C:\xampp\htdocs\protected\views\pieteikums\view.php:18)
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(870): PieteikumsController->renderFile()
#1 C:\xampp\htdocs\yii\web\CController.php(783): PieteikumsController->renderPartial()
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(41): PieteikumsController->render()
#3 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): PieteikumsController->actionView()
#4 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#6 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/view
in C:\xampp\htdocs\protected\views\pieteikums\view.php (18)
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (41)
in C:\xampp\htdocs\index.php (13)
2012/10/05 15:23:42 [error] [php] Undefined variable: k (C:\xampp\htdocs\protected\views\pieteikums\view.php:18)
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(870): PieteikumsController->renderFile()
#1 C:\xampp\htdocs\yii\web\CController.php(783): PieteikumsController->renderPartial()
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(41): PieteikumsController->render()
#3 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): PieteikumsController->actionView()
#4 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#6 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/view
in C:\xampp\htdocs\protected\views\pieteikums\view.php (18)
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (41)
in C:\xampp\htdocs\index.php (13)
2012/10/05 15:23:50 [error] [php] Undefined variable: k (C:\xampp\htdocs\protected\views\pieteikums\view.php:18)
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(870): PieteikumsController->renderFile()
#1 C:\xampp\htdocs\yii\web\CController.php(783): PieteikumsController->renderPartial()
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(41): PieteikumsController->render()
#3 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): PieteikumsController->actionView()
#4 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#6 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/view
in C:\xampp\htdocs\protected\views\pieteikums\view.php (18)
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (41)
in C:\xampp\htdocs\index.php (13)
2012/10/05 15:23:51 [error] [php] Undefined variable: k (C:\xampp\htdocs\protected\views\pieteikums\view.php:18)
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(870): PieteikumsController->renderFile()
#1 C:\xampp\htdocs\yii\web\CController.php(783): PieteikumsController->renderPartial()
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(41): PieteikumsController->render()
#3 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): PieteikumsController->actionView()
#4 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#6 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/view
in C:\xampp\htdocs\protected\views\pieteikums\view.php (18)
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (41)
in C:\xampp\htdocs\index.php (13)
2012/10/05 15:23:56 [error] [php] Undefined variable: k (C:\xampp\htdocs\protected\views\pieteikums\view.php:18)
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(870): PieteikumsController->renderFile()
#1 C:\xampp\htdocs\yii\web\CController.php(783): PieteikumsController->renderPartial()
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(41): PieteikumsController->render()
#3 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): PieteikumsController->actionView()
#4 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#6 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/view
in C:\xampp\htdocs\protected\views\pieteikums\view.php (18)
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (41)
in C:\xampp\htdocs\index.php (13)
2012/10/05 15:23:57 [error] [php] Undefined variable: k (C:\xampp\htdocs\protected\views\pieteikums\view.php:18)
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(870): PieteikumsController->renderFile()
#1 C:\xampp\htdocs\yii\web\CController.php(783): PieteikumsController->renderPartial()
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(41): PieteikumsController->render()
#3 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): PieteikumsController->actionView()
#4 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#6 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/view
in C:\xampp\htdocs\protected\views\pieteikums\view.php (18)
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (41)
in C:\xampp\htdocs\index.php (13)
2012/10/05 15:44:40 [warning] [application] Failed to set unsafe attribute "teksts" of "Pieteikumi".
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (21)
in C:\xampp\htdocs\index.php (13)
2012/10/05 16:03:13 [error] [php] Undefined variable: model (C:\xampp\htdocs\protected\views\pieteikums\decline.php:14)
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(870): PieteikumsController->renderFile()
#1 C:\xampp\htdocs\yii\web\CController.php(783): PieteikumsController->renderPartial()
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(49): PieteikumsController->render()
#3 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): PieteikumsController->actionDecline()
#4 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#6 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/decline&id=1
in C:\xampp\htdocs\protected\views\pieteikums\decline.php (14)
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (49)
in C:\xampp\htdocs\index.php (13)
2012/10/05 16:03:14 [error] [php] Undefined variable: model (C:\xampp\htdocs\protected\views\pieteikums\decline.php:14)
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(870): PieteikumsController->renderFile()
#1 C:\xampp\htdocs\yii\web\CController.php(783): PieteikumsController->renderPartial()
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(49): PieteikumsController->render()
#3 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): PieteikumsController->actionDecline()
#4 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#6 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/decline&id=1
in C:\xampp\htdocs\protected\views\pieteikums\decline.php (14)
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (49)
in C:\xampp\htdocs\index.php (13)
2012/10/05 16:07:55 [error] [php] Undefined variable: model (C:\xampp\htdocs\protected\views\pieteikums\decline.php:14)
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(870): PieteikumsController->renderFile()
#1 C:\xampp\htdocs\yii\web\CController.php(783): PieteikumsController->renderPartial()
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(49): PieteikumsController->render()
#3 unknown(0): PieteikumsController->actionDecline()
#4 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#8 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#10 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#11 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#12 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/decline&id=1
in C:\xampp\htdocs\protected\views\pieteikums\decline.php (14)
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (49)
in C:\xampp\htdocs\index.php (13)
2012/10/05 16:07:58 [error] [php] Undefined variable: model (C:\xampp\htdocs\protected\views\pieteikums\decline.php:14)
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(870): PieteikumsController->renderFile()
#1 C:\xampp\htdocs\yii\web\CController.php(783): PieteikumsController->renderPartial()
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(49): PieteikumsController->render()
#3 unknown(0): PieteikumsController->actionDecline()
#4 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#8 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#10 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#11 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#12 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/decline&id=1
in C:\xampp\htdocs\protected\views\pieteikums\decline.php (14)
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (49)
in C:\xampp\htdocs\index.php (13)
2012/10/05 16:08:05 [error] [php] Undefined variable: model (C:\xampp\htdocs\protected\views\pieteikums\decline.php:14)
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(870): PieteikumsController->renderFile()
#1 C:\xampp\htdocs\yii\web\CController.php(783): PieteikumsController->renderPartial()
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(49): PieteikumsController->render()
#3 unknown(0): PieteikumsController->actionDecline()
#4 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#8 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#10 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#11 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#12 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/decline&id=1
in C:\xampp\htdocs\protected\views\pieteikums\decline.php (14)
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (49)
in C:\xampp\htdocs\index.php (13)
2012/10/05 16:33:38 [error] [php] A non well formed numeric value encountered (C:\xampp\htdocs\protected\controllers\PieteikumsController.php:55)
Stack trace:
#0 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs()
#1 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CInlineAction->runWithParamsInternal()
#2 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#3 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#4 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#5 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#6 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#7 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/decline&id=1
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (55)
in C:\xampp\htdocs\index.php (13)
2012/10/05 16:33:40 [error] [php] A non well formed numeric value encountered (C:\xampp\htdocs\protected\controllers\PieteikumsController.php:55)
Stack trace:
#0 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs()
#1 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CInlineAction->runWithParamsInternal()
#2 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#3 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#4 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#5 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#6 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#7 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/decline&id=1
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (55)
in C:\xampp\htdocs\index.php (13)
2012/10/05 16:38:22 [error] [exception.CException] exception 'CException' with message 'Property "Pieteikumi.tableName" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:174
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(160): CComponent->__set('tableName', 'arhivs_pieteiku...')
#1 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(53): CActiveRecord->__set('tableName', 'arhivs_pieteiku...')
#2 [internal function]: PieteikumsController->actionDecline('2')
#3 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(PieteikumsController), Array)
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(PieteikumsController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('decline')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('pieteikums/decl...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=pieteikums/decline&id=2
HTTP_REFERER=http://localhost/index.php?r=pieteikums/view
---
2012/10/05 16:38:23 [error] [exception.CException] exception 'CException' with message 'Property "Pieteikumi.tableName" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:174
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(160): CComponent->__set('tableName', 'arhivs_pieteiku...')
#1 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(53): CActiveRecord->__set('tableName', 'arhivs_pieteiku...')
#2 [internal function]: PieteikumsController->actionDecline('2')
#3 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(PieteikumsController), Array)
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(PieteikumsController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('decline')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('pieteikums/decl...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=pieteikums/decline&id=2
HTTP_REFERER=http://localhost/index.php?r=pieteikums/decline&id=2
---
2012/10/05 16:41:05 [warning] [application] Failed to set unsafe attribute "id" of "ArhivsPieteikumi".
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (59)
in C:\xampp\htdocs\index.php (13)
2012/10/05 18:09:01 [error] [exception.CException] exception 'CException' with message 'PieteikumsController cannot find the requested view "accept".' in C:\xampp\htdocs\yii\web\CController.php:879
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(783): CController->renderPartial('accept', NULL, true)
#1 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(47): CController->render('accept')
#2 [internal function]: PieteikumsController->actionAccept('9')
#3 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(PieteikumsController), Array)
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(PieteikumsController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('accept')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('pieteikums/acce...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=pieteikums/accept&id=9
HTTP_REFERER=http://localhost/index.php?r=pieteikums/view
---
2012/10/05 18:09:02 [error] [exception.CException] exception 'CException' with message 'PieteikumsController cannot find the requested view "accept".' in C:\xampp\htdocs\yii\web\CController.php:879
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(783): CController->renderPartial('accept', NULL, true)
#1 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(47): CController->render('accept')
#2 [internal function]: PieteikumsController->actionAccept('9')
#3 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(PieteikumsController), Array)
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(PieteikumsController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('accept')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('pieteikums/acce...')
#10 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php?r=pieteikums/accept&id=9
HTTP_REFERER=http://localhost/index.php?r=pieteikums/accept&id=9
---
2012/10/05 18:21:20 [error] [exception.CException] exception 'CException' with message 'Property "Pieteikumi.uzvards" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(144): CComponent->__get('uzvards')
#1 C:\xampp\htdocs\yii\base\CModel.php(593): CActiveRecord->__get('uzvards')
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(53): CModel->offsetGet('uzvards')
#3 [internal function]: PieteikumsController->actionAccept('9')
#4 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(PieteikumsController), Array)
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(PieteikumsController), Object(ReflectionMethod), Array)
#6 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('accept')
#10 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('pieteikums/acce...')
#11 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#12 C:\xampp\htdocs\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/index.php?r=pieteikums/accept&id=9
HTTP_REFERER=http://localhost/index.php?r=pieteikums/view
---
2012/10/05 18:21:21 [error] [exception.CException] exception 'CException' with message 'Property "Pieteikumi.uzvards" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(144): CComponent->__get('uzvards')
#1 C:\xampp\htdocs\yii\base\CModel.php(593): CActiveRecord->__get('uzvards')
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(53): CModel->offsetGet('uzvards')
#3 [internal function]: PieteikumsController->actionAccept('9')
#4 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(PieteikumsController), Array)
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(PieteikumsController), Object(ReflectionMethod), Array)
#6 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('accept')
#10 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('pieteikums/acce...')
#11 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#12 C:\xampp\htdocs\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/index.php?r=pieteikums/accept&id=9
HTTP_REFERER=http://localhost/index.php?r=pieteikums/accept&id=9
---
2012/10/05 18:21:27 [error] [exception.CException] exception 'CException' with message 'Property "Pieteikumi.uzvards" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(144): CComponent->__get('uzvards')
#1 C:\xampp\htdocs\yii\base\CModel.php(593): CActiveRecord->__get('uzvards')
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(53): CModel->offsetGet('uzvards')
#3 [internal function]: PieteikumsController->actionAccept('9')
#4 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(PieteikumsController), Array)
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(PieteikumsController), Object(ReflectionMethod), Array)
#6 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('accept')
#10 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('pieteikums/acce...')
#11 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#12 C:\xampp\htdocs\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/index.php?r=pieteikums/accept&id=9
HTTP_REFERER=http://localhost/index.php?r=pieteikums/accept&id=9
---
2012/10/05 18:21:45 [error] [php] Undefined variable: pass (C:\xampp\htdocs\protected\views\pieteikums\accept.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(870): PieteikumsController->renderFile()
#1 C:\xampp\htdocs\yii\web\CController.php(783): PieteikumsController->renderPartial()
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(72): PieteikumsController->render()
#3 unknown(0): PieteikumsController->actionAccept()
#4 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#8 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#10 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#11 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#12 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/accept&id=9
in C:\xampp\htdocs\protected\views\pieteikums\accept.php (19)
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (72)
in C:\xampp\htdocs\index.php (13)
2012/10/05 18:22:43 [error] [php] Undefined variable: pass (C:\xampp\htdocs\protected\views\pieteikums\accept.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\web\CController.php(870): PieteikumsController->renderFile()
#1 C:\xampp\htdocs\yii\web\CController.php(783): PieteikumsController->renderPartial()
#2 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(72): PieteikumsController->render()
#3 unknown(0): PieteikumsController->actionAccept()
#4 C:\xampp\htdocs\yii\web\actions\CAction.php(107): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(48): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii\web\CController.php(287): PieteikumsController->runAction()
#8 C:\xampp\htdocs\yii\web\CController.php(266): PieteikumsController->runActionWithFilters()
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(283): PieteikumsController->run()
#10 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#11 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#12 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=pieteikums/accept&id=10
in C:\xampp\htdocs\protected\views\pieteikums\accept.php (19)
in C:\xampp\htdocs\protected\controllers\PieteikumsController.php (72)
in C:\xampp\htdocs\index.php (13)
2012/10/05 18:26:43 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\protected\components\UserIdentity.php:40)
Stack trace:
#0 C:\xampp\htdocs\yii\validators\CValidator.php(214): CInlineValidator->validateAttribute()
#1 C:\xampp\htdocs\yii\base\CModel.php(158): CInlineValidator->validate()
#2 C:\xampp\htdocs\protected\controllers\SiteController.php(145): LoginForm->validate()
#3 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#4 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\web\CController.php(287): SiteController->runAction()
#6 C:\xampp\htdocs\yii\web\CController.php(266): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\web\CWebApplication.php(283): SiteController->run()
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php?r=site/login
in C:\xampp\htdocs\protected\components\UserIdentity.php (40)
in C:\xampp\htdocs\protected\models\LoginForm.php (66)
in C:\xampp\htdocs\protected\controllers\SiteController.php (145)
2012/10/06 09:51:24 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Jums nav ties카bu apmekl휆t 큄o lapu' in C:\xampp\htdocs\protected\components\Controller.php:27
Stack trace:
#0 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(41): Controller->allowUser(3)
#1 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): PieteikumsController->actionView()
#2 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('view')
#6 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('pieteikums/view')
#7 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\xampp\htdocs\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/index.php?r=pieteikums/view
---
2012/10/06 09:51:24 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Jums nav ties카bu apmekl휆t 큄o lapu' in C:\xampp\htdocs\protected\components\Controller.php:27
Stack trace:
#0 C:\xampp\htdocs\protected\controllers\PieteikumsController.php(41): Controller->allowUser(3)
#1 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(50): PieteikumsController->actionView()
#2 C:\xampp\htdocs\yii\web\CController.php(309): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\web\CController.php(287): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 C:\xampp\htdocs\yii\web\CWebApplication.php(283): CController->run('view')
#6 C:\xampp\htdocs\yii\web\CWebApplication.php(142): CWebApplication->runController('pieteikums/view')
#7 C:\xampp\htdocs\yii\base\CApplication.php(162): CWebApplication->processRequest()
#8 C:\xampp\htdocs\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/index.php?r=pieteikums/view
HTTP_REFERER=http://localhost/index.php?r=pieteikums/view
---
